runtime/test/strategy-sequence-test.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * @license
+ * Copyright (c) 2018 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import Arc from '../arc.js';
+import Loader from '../loader.js';
+import Planner from '../planner.js';
+import {assert} from './chai-web.js';
+import Manifest from '../manifest.js';
+import Recipe from '../recipe/recipe.js';
+import ConvertConstraintsToConnections from '../strategies/convert-constraints-to-connections.js';
+import InitPopulation from '../strategies/init-population.js';
+import MapSlots from '../strategies/map-slots.js';
+import ResolveRecipe from '../strategies/resolve-recipe.js';
+import MatchParticleByVerb from '../strategies/match-particle-by-verb.js';
+import MatchRecipeByVerb from '../strategies/match-recipe-by-verb.js';
+import SearchTokensToParticles from '../strategies/search-tokens-to-particles.js';
+import GroupHandleConnections from '../strategies/group-handle-connections.js';
+import CombinedStrategy from '../strategies/combined-strategy.js';
+import CreateDescriptionHandle from '../strategies/create-description-handle.js';
+import FallbackFate from '../strategies/fallback-fate.js';
+import MessageChannel from '../message-channel.js';
```

Comment:
Seemed like a lot of imports, this one doesn't appear to be used, didn't check them all.

Reply:
Mostly these are the strategies, and I think they probably should be included even if not (yet) used, to make it easier to use them later.

Actually we probably should build a dynamic strategy importer that grabs everything in the directory, and use that everywhere.

I'll audit the remaining ones though.